### PR TITLE
New Dashboard Redirects

### PR DESCRIPTION
**Pull request by NikhilNanjappa**
- Old dashboard screens redirect to the new dashboard screen
- Successfully logging in takes you to `/apply` and not account anymore
- navigation links change based on a `enableNewApplicationDashboards` flag
- Fixed tests for all the above

### CODE REVIEW

File: controllers/user/login.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ router
                             if (LoginRateLimiter.hasConsumedPoints()) {
                                 await LoginRateLimiter.clearRateLimit();
                             }
-                            redirectUrlWithFallback(req, res, '/user');
+                            redirectUrlWithFallback(req, res, '/apply');
```
Comment (davidrapson) [inline controllers/user/login.js]: We need to put this behind a feature flag, this will redirect _everyone_ to apply, even in production

**Comment by davidrapson** [comment]
Great work so far, minor comment about the login redirect, but realised this is still work in progress.

File: controllers/apply/form-router-next/index.js [hunk 2]
```diff
@@ -50,24 +50,45 @@ function initFormRouter({
...
-        ];
+        if (features.enableNewApplicationDashboards) {
+            res.locals.userNavigationLinks = [
+                {
+                    url: `${req.baseUrl}/summary`,
+                    label: res.locals.copy.navigation.summary
+                },
+                {
+                    url: localify(req.i18n.getLocale())('/apply'),
+                    label: 'Latest application'
```
Comment (mattandrews) [inline controllers/apply/form-router-next/index.js]: These need Welsh translations, right?

Comment (davidrapson) [reply controllers/apply/form-router-next/index.js]: They will do, but we are OK to merge for now behind a flag

**Pull request merged**